rotkehlchen/data/all_assets.json

```diff
@@ -13451,6 +13451,66 @@
         "symbol": "sUSD",
         "type": "ethereum token"
     },
+    "sBTC": {
+        "coingecko": "sbtc",
+        "cryptocompare": "btc",
```

Comment:
Note: For cryptocompare it needs to be all caps. It may be a temporary workaround to have BTC and ETH below as cryptocompare's links so that we can have historical interest rates but would need to note it down somewhere that it's not the same.

Since checking the price of sBTC and BTC they differ quite a bit.

Reply:
Yeah I wasn't sure what to do but I figured this is better than having no price information at all and the prices theoretically should be pretty close to each other....

Reply:
Also fixed the cryptocompare values